### ISSUE TITLE
Syntax error in first example

### ISSUE THREAD

**Issue by doubleh1223** [MicrosoftDocs/OfficeDocs-Exchange#630]
The first example is missing an ending bracket here "[-EnableRoomMailboxAccount &lt;$true | $false&gt; [-RoomMailboxPassword (ConvertTo-SecureString -String '&lt;Password&gt;' -AsPlainText -Force)] [-ResetPasswordOnNextLogon &lt;$true | $false&gt;]" I think it needs an extra ending bracket after '-AsPlainText -Force)]' Hope this makes sense.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d413bc48-105e-4bb9-725c-7a96cee8cd47
* Version Independent ID: d00eecb4-b87a-75d1-8d57-2d89552be7f4
* Content: [Convert a mailbox in Exchange Server](https://docs.microsoft.com/en-us/exchange/recipients/user-mailboxes/convert-mailboxes?view=exchserver-2019)
* Content Source: [Exchange/ExchangeServer/recipients/user-mailboxes/convert-mailboxes.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/recipients/user-mailboxes/convert-mailboxes.md)
* Product: **exchange-server-it-pro**
* GitHub Login: @chrisda
* Microsoft Alias: **chrisda**

**Comment by doubleh1223** [comment]
Looking back the bracket might also be supposed to be here [-EnableRoomMailboxAccount <$true | $false> ]

**Comment by chrisda** [comment]
@doubleh1223, in those syntax statements, the square brackets merely indicate optional parameter and value pairs. Not terribly important, but I agree they should be correct. I only saw a missing "]" at the end of `[-EnableRoomMailboxAccount <$true | $false>`, which I fixed.

I didn't see any missing square brackets around `[-Password (ConvertTo-SecureString -String '<Password>' -AsPlainText -Force)]` or `[-RoomMailboxPassword (ConvertTo-SecureString -String '<Password>' -AsPlainText -Force)]`.

Thanks for the feedback.
